index.html

```diff
@@ -0,0 +1,6 @@
+<head>
+    <link rel="stylesheet" href="./style.css">
+</head>
```

Comment:
Great effort here, @codedit334 :wink: 
- [ ] I understand that you probably reduced the content of your `head` section probably because the project was a small one. However, it is important that to ensure that the `head` section is properly configured. This is also making your HTML semantically correct. It also has the potential of making your page SEO-friendly because your page will render well. Please try to update your `title`, and other `meta` tags.